### PR TITLE
Fix for nft balanceOf to be decreased after claim

### PR DESCRIPTION
**Pull request by folkyatina**

### CODE REVIEW

File: contracts/0.8.9/WithdrawalQueueBase.sol [hunk 1]
```diff
@@ -442,6 +442,7 @@ abstract contract WithdrawalQueueBase {
         if (_recipient == address(0)) _recipient = request.owner;
 
         request.claimed = true;
+        _getRequestsByOwner()[request.owner].remove(_requestId);
```
Comment (TheDZhon) [inline contracts/0.8.9/WithdrawalQueueBase.sol:445]: maybe it worth checking the return status of the `remove` call (the same was proposed in #633)

**Review by TheDZhon — APPROVED** [review]
:+1:

File: contracts/0.8.9/WithdrawalQueueBase.sol (hunk 1 above)
Comment (folkyatina) [reply contracts/0.8.9/WithdrawalQueueBase.sol:445]: Probably, it is.

**Comment by folkyatina** [comment]
Merged into #632

**Pull request closed without merging**